apps/script/generateSharedConstants.rb

```diff
@@ -9,6 +9,7 @@
 require 'active_support/core_ext/hash'
 require 'fileutils'
 require 'require_all'
+require 'ostruct'
```

Comment:
This was an implicit dependency before (we use OpenStruct below), but it was automatically pulled in for us by older versions of the `json` gem.